contracts/src/core/EigenDACertVerifier.sol

```diff
@@ -147,7 +147,7 @@ contract EigenDABlobVerifier is IEigenDABlobVerifier {
         VersionedBlobParams memory blobParams,
         SecurityThresholds memory securityThresholds
     ) external view {
```

Comment:
I think this function `verifyBlobSecurityParams` should also be renamed (in addition to the subsequent `verifyBlobSecurityParams` function)